internal/auth/authn/aapgateway.go

```diff
@@ -87,10 +102,21 @@ func (a AapGatewayAuth) ValidateToken(ctx context.Context, token string) error {
 	return err
 }
 
-func (a AapGatewayAuth) GetAuthConfig() common.AuthConfig {
-	return common.AuthConfig{
-		Type: common.AuthTypeAAP,
-		Url:  a.externalGatewayUrl,
+func (a AapGatewayAuth) GetAuthConfig() *api.AuthConfig {
+	providerType := string(api.AuthProviderInfoTypeAap)
+	providerName := string(api.AuthProviderInfoTypeAap)
```

Comment:
Ideally there should be a way to define a `displayName` also to the static providers.